plotUtils/reach/simps22/2016_reach/hpstr_config/kalSimpTuple_cfg.py

```diff
@@ -0,0 +1,195 @@
+
+#Tracker3DHits
+svthitsgbl.parameters["hitCollLcio"]    = 'RotatedHelicalTrackHits' 
+svthitsgbl.parameters["hitCollRoot"]    = 'RotatedHelicalTrackHits' 
+svthitsgbl.parameters["mcPartRelLcio"]    = 'RotatedHelicalTrackMCRelations'
+
+#Tracking
+track.parameters["debug"] = 0
+track.parameters["trkCollLcio"] = 'KalmanFullTracks'
+track.parameters["trkCollRoot"] = 'KalmanFullTracks'
+track.parameters["kinkRelCollLcio"] = ''
+track.parameters["trkRelCollLcio"] = 'KFTrackDataRelations'
+track.parameters["trkhitCollRoot"] = 'SiClustersOnTrack'
+track.parameters["hitFitsCollLcio"] = 'SVTFittedRawTrackerHits'
+
+#Only for detail studies
+#LT uncomment
+track.parameters["rawhitCollRoot"] = 'SVTRawHitsOnTrack_KF'
+
+#LT uncommented
+#if (not options.isData):
+#    track.parameters["truthTrackCollLcio"] = 'KalmanFullTracksToTruthTrackRelations'
+#    track.parameters["truthTrackCollRoot"] = 'Truth_KFTracks'
+
+#LT check if we need the b field or not -- version of HPS java
+# for Jess's files need to give it b-field
+
+track.parameters["bfield"] = bfield[str(options.year)]
+
+trackgbl.parameters["debug"] = 0
+trackgbl.parameters["trkCollLcio"] = 'GBLTracks'
+trackgbl.parameters["trkCollRoot"] = 'GBLTracks'
+trackgbl.parameters["kinkRelCollLcio"] = 'GBLKinkDataRelations'
+trackgbl.parameters["trkRelCollLcio"] = 'TrackDataRelations'
+trackgbl.parameters["trkhitCollRoot"] = 'RotatedHelicalOnTrackHits'
+trackgbl.parameters["hitFitsCollLcio"] = 'SVTFittedRawTrackerHits'
+
+#Only for detail studies
+#LT uncomment
+trackgbl.parameters["rawhitCollRoot"] = 'SVTRawHitsOnTrack'
+
+#LT uncommented
+#if (not options.isData):
+#    trackgbl.parameters["truthTrackCollLcio"] = 'GBLTracksToTruthTrackRelations'
+#    trackgbl.parameters["truthTrackCollRoot"] = 'Truth_GBLTracks'
+
+#LT check if we need the b field or not -- version of HPS java
+trackgbl.parameters["bfield"] = bfield[str(options.year)]
+
+#ECalData
+ecal_gbl.parameters["debug"] = 0
+ecal_gbl.parameters["hitCollLcio"] = 'EcalCalHits'
+ecal_gbl.parameters["hitCollRoot"] = 'RecoEcalHits'
+ecal_gbl.parameters["clusCollLcio"] = "EcalClustersCorr"
+ecal_gbl.parameters["clusCollRoot"] = "RecoEcalClusters"
+
+ecal_kf.parameters["debug"] = 0
+ecal_kf.parameters["hitCollLcio"] = 'EcalCalHits'
+ecal_kf.parameters["hitCollRoot"] = 'RecoEcalHits'
+ecal_kf.parameters["clusCollRoot"] = "RecoEcalClusters"
+ecal_kf.parameters["clusCollLcio"] = "EcalClustersCorr"
+
+#Vertex
+vtx.parameters["debug"] = 0
+vtx.parameters["vtxCollLcio"]    = 'UnconstrainedV0Vertices_KF'
+vtx.parameters["vtxCollRoot"]    = 'UnconstrainedV0Vertices_KF'
+vtx.parameters["partCollRoot"]   = 'ParticlesOnVertices_KF'
+vtx.parameters["kinkRelCollLcio"] = ''
+vtx.parameters["trkRelCollLcio"] = 'KFTrackDataRelations'
+
+vtxgbl.parameters["debug"] = 0
+vtxgbl.parameters["vtxCollLcio"]     = 'UnconstrainedV0Vertices'
+vtxgbl.parameters["vtxCollRoot"]     = 'UnconstrainedV0Vertices'
+vtxgbl.parameters["partCollRoot"]    = 'ParticlesOnVertices'
```

Comment:
This one doesn't have the fix to the name of the Particle/Vertex collections yet